cloud/shared/bin/lib/config_loader.py

```diff
@@ -45,20 +47,37 @@ def use_local_backend(self):
     def skip_confirmations(self):
         return os.getenv('SKIP_CONFIRMATIONS', False)
 
-    def load_config(self):
-        self._load_config()
+    def load_config(self, config_file):
+        self._load_config(config_file)
         return self.validate_config()
 
-    def _load_config(self):
+    def _get_config_values_from_sh_file(self, config_file):
+        ## 1. Export all variables from the config into clean environment
+        ## 2. Set values in current environment
+        if not os.path.exists(config_file):
+            exit(f'Cannot find file {config_file}')
+        print(f'Getting config from {config_file}')
+        command = shlex.split(f'env -i bash -c "source {config_file} && env"')
+        proc = subprocess.Popen(command, stdout=subprocess.PIPE)
+        for line in proc.stdout:
+            (key, _, value) = line.decode().partition("=")
+            os.environ[key] = value.strip()
```

Comment:
Hm. Do we actually need to expose these variables back to environment? Or can pass around as config?

Reply:
Currently ConfigLoader reads some values from env variables, and I don't want to change that in this PR.
Next big change is to make config yaml and all of this code will be changed. At that point ConfigLoader will be one place to get all the values.
It will likely require rewrite of some bash scripts that we still have.